druid/Dockerfile

```
@@ -59,9 +76,9 @@ rm -rf /stackable/apache-druid-${PRODUCT}-src
 # and we are sometimes running into errors because we're out of space.
 # Therefore, we try to clean up all layers as much as possible.
 if [ "${DELETE_CACHES}" = "true" ] ; then
-  rm -rf /stackable/.m2/repository/*
-  rm -rf /stackable/.npm/*
-  rm -rf /stackable/.cache/*
+rm -rf /stackable/.m2/repository/*
+rm -rf /stackable/.npm/*
+rm -rf /stackable/.cache/*
```

Revert the indentation:

```suggestion
  rm -rf /stackable/.m2/repository/*
  rm -rf /stackable/.npm/*
  rm -rf /stackable/.cache/*
```